app/models/partners/child.rb

```diff
@@ -26,7 +25,7 @@ class Child < Base
     serialize :child_lives_with, Array
     belongs_to :family
     has_many :child_item_requests, dependent: :destroy
-    has_and_belongs_to_many :needed_items, class_name: 'Item', join_table: :needed_items
+    has_and_belongs_to_many :needed_items, class_name: 'Item'
```

Comment:
This should still just be `has_many`, no?

Reply:
in that case `items` must have `child_id`. Or `has_many .... through`

Reply:
Yep, can we change to `has_many :through`? That's more accurate for the actual relationship here.

Reply:
Ah, took another look and you're right - this is a HABTM. I'm not used to using this relationship because it's rare that it's legitimately cross-join. 

Still have a question though - the table is now called `children_items` but the relationship is called `needed_items`... I don't actually see the table being referenced. Is there some Rails magic I'm missing?